docker/run_setup.sh

```diff
@@ -13,14 +13,16 @@ set -e
 # types, indexes, and keys; also bulk-loads static data for some lookup tables
 /app/docker/run_setup_postgres.sh
 
-# Drop and re-create local S3 buckets
+# Delete and create local S3 buckets
 /app/docker/run_recreate_s3_buckets.sh
 
-# Delete all Elasticsearch indices
+# Delete and create Elasticsearch indices
 /app/socorro-cmd clear_indices
+/app/socorro-cmd es create
 
-# Delete and re-create Pub/Sub topics and subscriptions
-/app/docker/run_setup_pubsub.sh
+# Delete and create Pub/Sub topics and subscriptions
+/app/socorro-cmd pubsub delete
+/app/socorro-cmd pubsub create
```

Comment:
The pubsub script only deleted and created things, so I just moved that here. With postgres and s3, it's a bit more involved. It might make sense to pull it all into this script at some point.